pkg/systemd/services/services.jsx

```diff
@@ -947,8 +974,8 @@
                              className="services-text-filter"
                              placeholder={_("Filter by name or description")}
                              value={currentTextFilter}
-                             onChange={setCurrentTextFilter}
-                             onClear={() => setCurrentTextFilter('')} />
+                             onChange={onTextFilterChanged}
+                             onClear={() => onTextFilterChanged('')} />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18115-20230103-122827-2f1a70ab-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)